vertical-pod-autoscaler/pkg/admission-controller/logic/server.go

```diff
@@ -178,16 +178,22 @@ func validateVPA(vpa *vpa_types.VerticalPodAutoscaler) error {
 		}
 	}
 
+	if isCreate {
```

Comment:
You can fold the two ifs into one.

Reply:
Sorry, I started with a more complicated code and haven't cleaned this. Fixed.